python/llm/test/benchmark/check_html_lines.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Python program to check if the number of lines in html meets expectation
+
+import os
+import sys
+import argparse
+import pandas as pd
+
+def main():
+    parser = argparse.ArgumentParser(description="check if the number of lines in html meets expectation")
+    parser.add_argument("-f", "--folder_path", type=str, dest="folder_path",
```

Comment:
I think you only need to input one file?
And you can directly use the raw csv file to do so in the original folder?

Reply:
ok. Will change this according to your suggestions.